Gemfile

```diff
@@ -145,6 +145,10 @@ group :embedded_terraform, :manageiq_default do
   manageiq_plugin "manageiq-providers-embedded_terraform"
 end
 
+group :nutanix, :manageiq_default do
+  manageiq_plugin "manageiq-providers-nutanix"
+end
+
```

Comment:
I don't want to merge this yet, until it's more "ready".  Can we split out the vendor change?  (Actually didn't nutanix try to dynamically update the vendor)?

Reply:
Yeah they dropped that at one point though, I think they just added it to this list locally
Okay I removed adding this to the Gemfile